src/blocks/select/Select.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useRef, useState, ReactNode, useEffect } from 'react';
+import React, { useRef, useState, ReactNode, useEffect, useCallback } from 'react';
```

Comment:
@mishramonalisha76 there's a bug in the Select component.

- Open the Select component.
- Let's say the list of the  popover is quite large then you have scroll inside the popover. When you try to scroll to the bottom of the list the popover closed.